qemu.confd

```diff
@@ -16,6 +16,9 @@
 # suitable for you, then you can omit that variable in your VM config.
 #
 
+# Name of virtual machine, by default /etc/init.d/qemu.<VM_NAME>
+# Enable when you change variables containing it.
+#VM_NAME="${RC_SVCNAME#qemu.}"
```

Comment:
This will not work, `VM_NAME` is not defined to be overridable by the confd file.